kratos/solving_strategies/strategies/butcher_tableau.h

```diff
@@ -241,15 +243,15 @@ class ButcherTableauForwardEuler : public ButcherTableau<ButcherTableauForwardEu
 
     static const BaseType::VectorType GenerateWeights()
     {
-        VectorType B;
-        B(0) = 1.0;
+        VectorType B(1);
+        B[0] = 1.0;
         return B;
     }
 
     static const BaseType::VectorType GenerateThetasVector()
     {
-        VectorType C;
-        C(0) = 0.0;
+        VectorType C(1);
+        C[0] = 0.0;
```

Comment:
```suggestion
        VectorType C(1);
```

this gets default constructed!

Reply:
I know, but I want to make clear which is the coefficient being used.